### ISSUE TITLE
function forward without function implementation triggers No RETURN statement in function

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#593]
Since update to 2.20 with antlr4 (from 2.13):

```
FUNCTION foo RETURNS LOGICAL PRIVATE (
) FORWARD.
FUNCTION bar RETURNS LOGICAL PRIVATE (
) FORWARD.


FUNCTION bar RETURNS LOGICAL PRIVATE (
):
   RETURN TRUE.
END FUNCTION.

RETURN "0":u.
```
![image](https://user-images.githubusercontent.com/5297464/46947745-24663400-d07c-11e8-9015-bc5e3071ab7d.png)

**Comment by gquerret** [comment]
Fixed issue for this case. But shouldn't lack of implementation raise an issue ?

**Comment by stefandrissen** [comment]
Yes, a forward that is not IN a handle to something else should raise an issue. 

These function forwards without implementation were an oversight.

**Comment by gquerret** [comment]
I'll create another ticket for this new rule (ended up to be an useful bug !).
And I'll just verify one last thing before closing this issue and merging into develop branch.

**Comment by gquerret** [comment]
Fixed in master
